src/components/DocumentView/Embed.tsx

```diff
@@ -11,6 +12,8 @@ import { IntegrationBlock } from './Integration';
 
 export async function Embed(props: BlockProps<gitbookAPI.DocumentBlockEmbed>) {
     const { block, context, ...otherProps } = props;
+    
+    ReactDOM.preconnect('https://cdn.iframe.ly');
```

Comment:
nit: Maybe define a constant like we have for `assetsDomain`?

Reply:
Why ?